src/Models/Attribute.php

```diff
@@ -12,11 +13,20 @@ class Attribute extends Model
 
     protected $primaryKey = 'attribute_id';
 
+    protected $appends = ['filterable'];
+
     protected static function booting()
     {
         static::addGlobalScope(new OnlyProductAttributesScope);
     }
 
+    protected function filterable(): CastsAttribute
+    {
+        return CastsAttribute::make(
+            get: fn () => $this->filter || in_array($this->code, config('rapidez.extra-filters')),
+        )->shouldCache();
+    }
```

Comment:
We can just use an accessor on `filter`: https://laravel.com/docs/master/eloquent-mutators#defining-an-accessor